helm/cloud-provider-vsphere/charts/kube-vip-cloud-provider/values.yaml

```diff
@@ -0,0 +1,39 @@
+# Default values for kube-vip-cloud-provider.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicasCount: 1
+
+image:
+  repository: kubevip/kube-vip-cloud-provider
```

Comment:
Don't we need to use retagged image?

Reply:
I forgot to replace the images. Thanks for reminding me. Here is the PR in retagger. https://github.com/giantswarm/retagger/pull/821